internal/cmd/invoice.go

```diff
@@ -0,0 +1,12 @@
+package cmd
+
+import "github.com/spf13/cobra"
+
+func init() {
+	rootCmd.AddCommand(invoiceCmd)
```

Comment:
I suggest avoiding cluttering the global cmd with this and adding it to `turso org`, making it a sibling of `turso org billing`.